corehq/motech/utils.py

```diff
@@ -59,18 +63,22 @@ def b64_aes_decrypt(message):
     Uses Django SECRET_KEY as AES key.
 
     >>> settings.SECRET_KEY = 'xyzzy'
```

Comment:
This feels like explaining a dad joke, which, surely, there are laws against. ... But this small audience might enjoy [the context](https://en.wikipedia.org/wiki/Xyzzy_(computing)).

```
$ adventure

Welcome to Adventure!!  Would you like instructions?
n

You are standing at the end of a road before a small brick building.
Around you is a forest.  A small stream flows out of the building and
down a gully.
```